### PR TITLE
Lp 47 back home btn

### LINKED ISSUE #47: Back to home button
**Is your feature request related to a problem? Please describe.**
There is no button to go back to the 

**Describe the solution you'd like**
To have a button that sends you back to the home page when you enter one of the categories pages

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. Added a go back to the home page button

Closes #47

### CODE REVIEW

File: modern-news-app/src/components/Main.tsx [hunk 1]
```diff
@@ -152,7 +189,18 @@ const Main: React.FC<MainProps> = ({ newsData }) => {
...
+                            <img 
+                                className="back-button"
+                                src={back} 
+                                alt="back arrow button to click and go bac to the home page" 
+                                onClick={() => setNewsData(new EverythingNewsResponse())} 
+                            />
+                            <div className="back-message">Go Back</div>
+                        </div>
+                        <TitleStyled style={{ color : `${mainColorCheck()}`}}>{searchTitleCheck()}</TitleStyled>
+                    </TitleSection>
```
Comment (JorgeLVilla) [inline modern-news-app/src/components/Main.tsx:203]: Cleeeean.

**Review by JorgeLVilla — APPROVED** [review]
Looks great Luis

**Review by BlackCubes — APPROVED** [review]
Looks gewd!

File: modern-news-app/src/components/Main.tsx [hunk 2]
```diff
@@ -96,11 +121,23 @@ const EnlargedArticle = styled.div<IEnlargeArticleProps>`
     }
 `
 
+export const ArticlesStyled = styled.div`
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
```
Comment (krferrell) [inline modern-news-app/src/components/Main.tsx:127]: There is a shorthand for this called flex-flow, like this "flex-flow: row wrap;"

**Review by krferrell — APPROVED** [review]
Look great Luis!

**Review by Gjhnsn — APPROVED** [review]
Looks great!

File: modern-news-app/src/components/Main.tsx (hunk 2 above)
Comment (Lap343) [reply modern-news-app/src/components/Main.tsx:127]: oooh thank you. Always just did it this way

**Pull request merged**